### ISSUE TITLE
Can you give me some advice on training the model？

### ISSUE THREAD

**Issue by BayMaxBHL** [SysCV/P3Depth#6]
After setting up the environment, I used NYU dataset for training, but the training results were very strange. The loss function converged slowly, rmse kept increasing, and delta kept decreasing.

**Comment by BayMaxBHL** [comment]
![image](https://user-images.githubusercontent.com/34785944/200978653-4ea1971c-17ac-44c0-af82-962e66e324fe.png)

**Comment by BayMaxBHL** [comment]
![image](https://user-images.githubusercontent.com/34785944/200978803-62db88e5-4cb8-4fe5-a88f-644076009c4a.png)

**Comment by BayMaxBHL** [comment]
![image](https://user-images.githubusercontent.com/34785944/200978863-9b62696b-3707-4ca8-90df-1a1e9c0bc9df.png)

**Comment by BayMaxBHL** [comment]
![image](https://user-images.githubusercontent.com/34785944/200978911-98a0861f-8eda-47e5-9347-7c4d784bd17f.png)

**Comment by BayMaxBHL** [comment]
![image](https://user-images.githubusercontent.com/34785944/200979062-1f123a28-9cfa-40ad-aacd-879cdde30307.png)

**Comment by BayMaxBHL** [comment]
The only changes I made to the code were to write a script to generate the CSV with a batch-size of 8.

**Comment by haifengwu205** [comment]
> After setting up the environment, I used NYU dataset for training, but the training results were very strange. The loss function converged slowly, rmse kept increasing, and delta kept decreasing.
Can you share your code, I can't run it with the current

**Comment by haifengwu205** [comment]
@BayMaxBHL Hello，can you run the test code? I run it with a size mismatch, as follows:
size mismatch for net.coords: copying a param with shape torch.Size([8, 480, 640, 2]) from checkpoint, the shape in current model is torch.Size([16, 480, 640, 2]).

**Comment by BayMaxBHL** [comment]
@haifengwu205 确实是用不了，我这不晒出来的结果就是不收敛嘛。rmse还卡卡往上涨，人都麻了。

**Comment by hutingz** [comment]
> @haifengwu205 确实是用不了，我这不晒出来的结果就是不收敛嘛。rmse还卡卡往上涨，人都麻了。

请问这个代码用的是多大数据的NYU呀

**Comment by macromogic** [comment]
Hi. I'm also trying to train the model myself and it does not converge as well. Does anyone have some solution so far? Thanks in advance!

**Comment by zhaorui-tan** [comment]
It's a similar situation to me. I used NYU dataset, and the result crashed as hell.  It seems the model learned a completed wrong thing. 
![image](https://github.com/SysCV/P3Depth/assets/53689438/10ed0a6d-52f5-4be3-b9ee-604439444d12)
![1697630602121](https://github.com/SysCV/P3Depth/assets/53689438/f9e5aeba-3b9a-4abb-8ac2-8ee9ee04e5e5)
![1697630674325](https://github.com/SysCV/P3Depth/assets/53689438/983370d3-c6bf-4429-9ce1-72b72c30752b)

**Comment by WJQ-NCHK** [comment]
Why did I not output any results after using the train.sh script?
